demo/guidance_test/write_simple_test.py

```diff
@@ -0,0 +1,68 @@
+import json
```

Comment:
I don't think this really needs to be here. It's used on our other repo to create these files, but I don't think it actually needs to be here. In addition, this is an old version.